speech-to-text/src/main/java/com/ibm/watson/developer_cloud/speech_to_text/v1/model/RecognizeOptions.java

```diff
@@ -136,6 +138,21 @@ public Builder speakerLabels(Boolean speakerLabels) {
     }
 
     /**
+     * If a custom model is specified, this parameter tells the service how much weight to give 
```

Comment:
Our code style check in Travis is failing because of trailing spaces in this line.